tests/datastore/test_base.py

```diff
@@ -6,3 +7,10 @@ def test_http_fs_parquet_as_df():
         "https://s3.wasabisys.com/iguazio/data/market-palce/aggregate/metrics.pq"
     )
     data_item.as_df()
+
```

Comment:
Based on the above, am I correct in assuming that storage_options are available as environmental variables?

Reply:
I don't think so, note this is not an s3 file path, but an http path, to a file in s3, it does not require any auth to get it.

Reply:
I've switched the test file for validating mlrun store objects will load into a Dask DataFrame a public container on Azure Open Datasets.  Is this acceptable?